### PR TITLE
fix(ngListDiretive): add ngListNoSeparator to split list by no separator

### PR DESCRIPTION
**Pull request by m-amr**

### CODE REVIEW

**Comment by gkalpak** [comment]
Fixes #13714.

**Comment by m-amr** [comment]
@Narretz I have updated the documentation, please check it.
Thanks.

File: src/ng/directive/ngList.js [hunk 1]
```diff
@@ -19,6 +20,12 @@
  *   when joining the list items back together) and whitespace around each list item is stripped
  *   before it is added to the model.
  *
+ * The The behaviour of the directive is affected by the use of the `ngListNoSeparator` attribute.
```
Comment (shoja) [inline src/ng/directive/ngList.js]: "The The" to "The"

**Comment by Narretz** [comment]
See here why we are not going to implement this: https://github.com/angular/angular.js/issues/13714#issuecomment-225170719

**Pull request closed without merging**